core/modules/system/system.admin.inc

```diff
@@ -695,14 +715,28 @@ function system_sort_themes($a, $b) {
  * Build a table row for the system modules page.
  */
 function _system_modules_build_row($info, $extra) {
+  // Prepare tag list by assureing all are lower case.
```

Comment:
```suggestion
  // Prepare the tag list by ensuring all tags are lower case.
```

Reply:
...btw, I don't see how the code below makes sure that tags are lower-cased.

Reply:
...I don't see how the code below makes it so that tags are lower-cased btw(?).

Reply:
Fixed.